azure-functions/acbs-function/unit.ff.jest.config.js

```diff
@@ -0,0 +1,8 @@
+const commonSettings = require('./jest.common.config');
```

Comment:
can we move towards using import and export? ES6

Reply:
Yes good shout, can I do this in a new PR so I can refactor all these files otherwise I fear this PR might get cumbersome to review?